python/plugins/processing/gui/Postprocessing.py

```diff
@@ -276,11 +281,17 @@ def handleAlgorithmResults(
         current_selected_node = iface.layerTreeView().currentNode()
         iface.layerTreeView().setUpdatesEnabled(False)
 
-    # store the current intersection point to restore it later
-    previous_insertion_point = (
-        details.project.layerTreeRegistryBridge().layerInsertionPoint()
-    )
-    for group, layer_node in sorted_layer_tree_layers:
+    for group, layer_node, project in sorted_layer_tree_layers:
+        if not project:
+            project = context.project()
+
+        # store the current intersection point to restore it later
```

Comment:
Typo "intersection" -> "insertion"

```suggestion
        # store the current insertion point to restore it later
```